addons/web/static/src/views/form/form_controller.scss

```
@@ -857,6 +857,16 @@
         padding-right: 0 !important;
     }
 
+    //specific for styling labels and checkboxes from product_views.xml
+    .o_product_view {
+        .o_field_boolean {
+            margin-right: 0;
+        }
+        .o_form_label {
```

Add an empty row before the selector

```suggestion

        .o_form_label {
```

---

I changed the selector in the latest commit